guides/browser-support.md

```diff
@@ -0,0 +1,171 @@
+returns a response, the default behavior would be to reload the page (or
+navigate to a response specific page) with the server's response. To improve
+the experience, we could submit the form via an AJAX request and inject the
+response into the page. We could then remove the logic to handle the servers
+response on page load to reduce our own technical debt, but what about users
+that can't access the scripting? Rather than creating a broken experience, we
+ensure the default behavior continues to work uninpeeded, and then build upon
+it to improve the experience for the browsers that can handle it.
+
+#### Known feature differences
+
+- JavaScript: We do not serve interactive scripting to IE 8 but we do deliver
+  analytics via JavaScript.
+- Icons: We currently use icon fonts to deliver scalable icons. Browsers that
+  do not support icon fonts unfortunately do not receive backups but we try to
+  always pair icons with text to ensure a user can continue to interpret a
+  page's content.
+
+### Automation
+
+Thankfully the world has created tools to help us track and polyfill
+disparities between browsers. We rely on a handful of of these tools to help
+us provide a level of backward compatability for modern features. As mentioned
+above, this doesn't necessarily mean feature parity. Where it's impossible or
+impractical to implement a modern feature, we fallback to standard practices
+for that browser. For example, we do not deliver interactive scripting for
+Internet Explorer 8, but we do ensure that default browser features continue to
+work so users that can't or don't want to upgrade continue to have access to
+the site and our content.
+
+#### Configuration
+
+Each project has it's own configuration depending on the requirements of the
+tools and features that project depends on. As a general rule we strive to
+cover the following browser list in each project.
+
+We configure [Autoprefixer](#autoprefixer) and [Babel](#babel) to support
+the following list of browsers.
+
+- Latest 2 releases of all browsers including:
+    - Chrome
+    - Firefox
+    - Safari
+    - Internet Explorer
+    - Edge
+    - Opera
+    - iOS Safari
+    - Opera Mini
+    - Android Browser
+    - BlackBerry Browser
+    - Opera Mobile
+    - Chrome for Android
+    - Firefox for Android
+    - Samsung Internet
+- Internet Explorer 9
+
+http://browserl.ist/?q=last+2+versions%2C+Explorer+%3E%3D+9
+
+As well as additional Autoprefixer support for:
+
+- Internet Explorer 8
+
+http://browserl.ist/?q=last+2+versions%2C+Explorer+%3E%3D+8
+
+### Browser Testing
+
+We use automated tests with a headless version of Chrome to ensure the majority
+of the site works as expected. For manual testing, we realistically test this
+project locally or in a virtual environment with the following list of
+browsers:
+
+- Chrome
+- Firefox
+- Safari
+- Internet Explorer 8, 9, 10, and 11
+- Edge
+- iOS Safari
+- Chrome for Android
+
+#### Individual projects to test
+
```

Comment:
Add a description about using each project's browser list when developing or editing features.

Reply:
Remember to do this.